rules/do-you-send-control4-notifications-to-a-slack-channel/rule.md

```diff
@@ -13,6 +13,37 @@ redirects: []
 
 ---
 
-Enter Intro Text
+Are you tired of missing important Control4 notifications? Fear not! We have a solution for you. By using the "Incoming Webhook" integration in Slack, you can receive Control4 notifications directly in Slack. 📣💬
 
 <!--endintro-->
+
+To set up Control4 notifications in Slack, follow these steps:
+
+1. Open Slack and go to your desired channel or create a new one dedicated to Control4 notifications. 📢
+2. Click on the channel name and select "Add apps" from the dropdown menu. 🔌
+3. Search for "Incoming Webhooks" and click on the app to install it. 
+4. Configure the webhook by providing a name for it and selecting the channel where you want the notifications to appear. 📝
+5. Copy the generated webhook URL. This will be used to send Control4 notifications to Slack. 
+
+::: good
+![Figure: Good example - Slack Integration window](slackintegration.jpg)
```

Comment:
Hey @kikibianc 

You need to upload this and the other 2 images to the rule folder

Reply:
Nevermind... I saw you added them in a separated PR: https://github.com/SSWConsulting/SSW.Rules.Content/pull/6144